src/main/java/org/team199/robot2022/commands/RetractClimber.java

```diff
@@ -9,24 +9,23 @@
 import edu.wpi.first.wpilibj2.command.FunctionalCommand;
 import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
 
-public class RetractClimber extends ParallelCommandGroup {
+public final class RetractClimber extends ParallelCommandGroup {
 
     public RetractClimber(Climber climber) {
         super(
             new FunctionalCommand(
-                () -> {},
-                climber::retractLeft,
-                climber::stopLeft,
-                climber::isLeftRetracted
+              () -> {},
+              () -> {climber.moveMotors(Climber.MotorSpeed.retract,climber.rightMotor);},
+              (interrupted) -> {climber.stopMotors(climber.rightMotor);},
+              () -> {return climber.isMotorRetracted(climber.rightMotor);}
             ),
             new FunctionalCommand(
-                () -> {},
-                climber::retractRight,
-                climber::stopRight,
-                climber::isRightRetracted
+              () -> {},
+              () -> {climber.moveMotors(Climber.MotorSpeed.retract,climber.leftMotor);},
+              (interrupted) -> {climber.stopMotors(climber.leftMotor);},
+              () -> {return climber.isMotorRetracted(climber.leftMotor);}
```

Comment:
```suggestion
              () -> {climber.moveMotors(Climber.MotorSpeed.retract,Climber.leftMotor);},
              (interrupted) -> {climber.stopMotors(Climber.leftMotor);},
              () -> {return climber.isMotorRetracted(Climber.leftMotor);}
```
Static members should be accessed in a static manner. (i.e. use `Climber` instead of `climber`)